### DISCUSSION TITLE
SSL connection timeout while trying to run a command

### DISCUSSION THREAD

**Discussion by wemmy6673** [Cyfrin/foundry-full-course-cu#3001] [Q&A]
I tried running this command  "curl -L https://foundry.paradigm.xyz | bash" on my terminal and i'm getting "SSL connection timeout ". Meanwhile my internet connection is good. How do i go about this?

**Comment by EngrPips** [comment]
Yeah, your connection might be good but unstable, I think if you retry when your connection is more stable things should work correctly.

**Comment by RaphDeAnalyst** [comment]
I encountered the same problem and have been on it for two days now, trying a lot of solutions. But using a VPN solved it perfectly, in my case I just downloaded Hotspotshield VPN and everything went well.

**Reply by EngrPips** [reply]
Awesome, Thanks for the heads-up buddy.
